### DISCUSSION TITLE
E-Commerce

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#125] [Q&A]
Since Slatwall went to a pay model, how is everyone integrating any sort of commerce into their Masa sites?

**Comment by Tigerlady** [comment]
I haven't yet, but I have an old shopping cart code that was once a plugin on MURA.  I am looking for some volunteers to update the code.  Some of it is pretty old and needs a refresh.  I would like someone who is very good at Coldfusion. I can do the CSS, but it is not a bad shopping cart option and is customizable to fit needs.  If interested PM me and I can give you more details.

**Reply by brucebailey** [reply]
Sent you an email, thank you!
